client/src/component/admin/MenuListContainer.js

```diff
@@ -15,7 +15,7 @@ import MenuList from "./MenuList";
 import MenuForm from "./MenuForm";
 import { AdminContext } from "./AdminPage";
 
-function MenuListContainer() {
+export function MenuListContainer() {
```

Comment:
I found out the api test wasn't working because you were testing the authentication wrapped component, and since you didn't mock authentication, it was actually working as expected and not rendering the component.

https://github.com/priyaraj7/guruprasad/blob/6fc6b9a8402ec6c22c526b3cc37e9c017875bbe7/client/src/component/admin/MenuListContainer.js#L153

So I'm exporting this one for testing.